src/random_test.cpp

```diff
@@ -112,6 +113,25 @@ int sum_of_all_options(Thd1 *thd) {
     opt_int_set(ALTER_REDO_LOGGING, 0);
   }
 
+  static auto lock = opt_string(LOCK);
```

Comment:
no need to define static. since we are not going to use it.

Reply:
fixed